src/components/TransactionFormInput.js

```diff
@@ -9,64 +9,77 @@ TransactionFormInput.propTypes = {
   onSave: PropTypes.func.isRequired,
 }
 
-export default function TransactionFormInput({ onSave }) {
+export default function TransactionFormInput({
+  onSave,
+  notes = '',
+  value = '',
+  tag = '',
+}) {
   const history = useHistory()
+
   return (
     <Formik
-      initialValues={{ value: '', notes: '', tag: '' }}
+      initialValues={{ value, notes, tag }}
+      enableReinitialize={true}
       validationSchema={Yup.object().shape({
         value: Yup.string()
           .required('Required')
           .min(1, 'Please enter a value')
-          .matches(
-            /^[0-9]+([.,][0-9]{1,2})?$/,
-            'Invalid format. Examples: 1200,34'
-          ),
+          .matches(/^[0-9]+([.,][0-9]{1,2})?$/, 'Invalid format.'),
         notes: Yup.string().max(100, 'Too long'),
         tag: Yup.string().required('Required'),
       })}
       onSubmit={(form, { resetForm }) => {
         onSave(form.value, form.notes, form.tag)
-        resetForm({ values: '' })
+        resetForm({
+          values: { value, notes, tag },
+        })
         history.push('/')
       }}
     >
-      {({ errors, touched }) => (
-        <FormStyled>
-          <Input name="value" placeholder="Enter transaction. Format: 20,00" />
-          {errors.value && touched.value ? (
-            <ErrorMessageInputStyled>{errors.value}</ErrorMessageInputStyled>
-          ) : null}
-          <LableStyled htmlFor="value">Euro</LableStyled>
-          <DropDown name="tag" component="select">
-            <option value="" selcted hidden>
-              -- Choose a tag --
-            </option>
-            <option value="Food">Food</option>
-            <option value="Leisure">Leisure</option>
-            <option value="FixedCosts">FixedCosts</option>
-            <option value="Transportation">Transportation</option>
-            <option value="Miscellaneous">Miscellaneous</option>
-            <option value="Earnings">Earnings</option>
-          </DropDown>
+      {({ values, errors, touched }) => {
+        return (
+          <FormStyled>
+            <Input
+              name="value"
+              placeholder="Enter transaction."
+              value={values.value}
```

Comment:
I find the naming of your cariables here a little bit confusing. Maybe you be more specific, what kind of value this is?

Reply:
Unfortunately, this is a built in functionality in Formik. It ensures that you can actually update the imported values